src/ducks/Studio/Chart/ContextMenu.js

```diff
@@ -55,6 +62,7 @@ export default ({ MetricNode, setOptions, onDeleteChartClick, ...props }) => {
       onCartesianGridChange={toggleCartesianGrid}
       onClosestDataChange={toggleClosestData}
       onWatermarkLighterChange={toggleWatermarkLighter}
+      onToggleWatermark={toggleShowWatermark}
```

Comment:
Inconsistent naming style. Better to name it`onWatermarkVisibilityChange` – `on[Target][Event]`